apidoc/Titanium/UI/TableView.yml

```diff
@@ -580,10 +580,28 @@ events:
         summary: Y coordinate of the event from the `source` view's coordinate system.
         type: Number
 
+      - name: force
+        summary: The current force value of the touch event.
+        type: Number
+
+      - name: maximumPossibleForce
+        summary: The maximum possible value , the force property can be.
+        type: Number
+
+      - name: altitudeAngle
+        summary: |
+            A value which indicates the stylus angle on the screen. If the stylus is perpendicular to the screen
+            or no stylus is being used, the value will be Pi/2. If the stylus is parallel to the screen the value will be 0.
+        type: Number
+
+      - name: timeStamp
+        summary: The time in seconds . of when the touch was used in corolation with the system start up.
+        type: Number
```

Comment:
Replace summary with `The time in seconds, of when the touch was used, in correlation with the system startup.`